dev/src/reference.ts

```diff
@@ -3342,53 +3393,93 @@ export class AggregateQuery<
    *
    * @private
    * @internal
-   * @param {bytes=} transactionId A transaction ID.
+   * @param transactionOrReadTime A transaction ID, options to start a new
+   *  transaction, or timestamp to use as read time.
    */
-  _get(
+  async _get(
     transactionIdOrReadTime?: Uint8Array | Timestamp
   ): Promise<
     AggregateQuerySnapshot<AggregateSpecType, AppModelType, DbModelType>
+  > {
+    const resp = await this._getResponse(transactionIdOrReadTime);
+    return resp.result;
+  }
+
+  /**
+   * Internal get() method that accepts an optional transaction id, and returns
+   * transaction metadata.
+   *
+   * @private
+   * @internal
+   * @param transactionOrReadTime A transaction ID, options to start a new
+   *  transaction, or timestamp to use as read time.
+   */
+  _getResponse(
+    transactionOrReadTime?: Uint8Array | Timestamp | api.ITransactionOptions
+  ): Promise<
+    QueryResponse<
+      AggregateQuerySnapshot<AggregateSpecType, AppModelType, DbModelType>
+    >
   > {
     // Capture the error stack to preserve stack tracing across async calls.
     const stack = Error().stack!;
 
     return new Promise((resolve, reject) => {
-      const stream = this._stream(transactionIdOrReadTime);
+      let responseTransaction: Uint8Array | undefined;
+      const stream = this._stream(transactionOrReadTime);
       stream.on('error', err => {
         reject(wrapError(err, stack));
       });
       stream.once('data', result => {
```

Comment:
Integration tests are fixed by changing this:

```suggestion
      stream.on('data', result => {
```